litmus-portal/cluster-k8s-manifest.yml

```diff
@@ -1,10 +1,458 @@
+
+  # for tracking & getting logs of the pods created by workflow controller to implement individual steps in the workflow
+  - apiGroups: [""]
+    resources: [pods/log]
+    verbs: [get, watch]
+
+  # for creation & deletion of application in predefined workflows
+  - apiGroups: [apps]
+    resources: [deployments, statefulsets]
+    verbs: [get, watch, patch , create, delete]
+    
+  # for creation, status polling and deletion of litmus chaos resources used within a chaos workflow
+  - apiGroups: [litmuschaos.io]
+    resources: [chaosengines, chaosexperiments, chaosresults, chaosschedules]
+    verbs: [create, list, get, patch, delete, watch]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: chaos-crb-for-litmusportal-server
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: chaos-cr-for-litmusportal-server
+subjects:
+  - kind: ServiceAccount
+    name: litmus-server-account
+    namespace: litmus
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: subscriber-cr-for-litmusportal-server
+  namespace: litmus
+  labels:
+    name: subscriber-cr-for-litmusportal-server
+rules:
+- apiGroups: [""]
+  resources: [configmaps]
+  verbs: [get, create, delete, update]
+- apiGroups: [""]
+  resources: [pods/log]
+  verbs: [get, list, watch]
+- apiGroups: [""]
+  resources: [pods, namespaces, nodes, services]
+  verbs: [get, list, watch]
+- apiGroups: [litmuschaos.io]
+  resources: [chaosengines, chaosschedules, chaosresults]
+  verbs: [get, list, create, delete, update, watch]
+- apiGroups: [apps.openshift.io]
+  resources: [deploymentconfigs]
+  verbs: [get, list]
+- apiGroups: [apps]
+  resources: [deployments, daemonsets, replicasets, statefulsets]
+  verbs: [get, list]
+- apiGroups: [argoproj.io]
+  resources: [workflows, workflows/finalizers, workflowtemplates, workflowtemplates/finalizers, cronworkflows, cronworkflows/finalizers, clusterworkflowtemplates, clusterworkflowtemplates/finalizers, rollouts]
+  verbs: [get, list, create, delete, update, watch]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: subscriber-crb-for-litmusportal-server
+  namespace: litmus
+subjects:
+  - kind: ServiceAccount
+    name: litmus-server-account
+    namespace: litmus
+roleRef:
+  kind: ClusterRole
+  name: subscriber-cr-for-litmusportal-server
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: event-tracker-cr-for-litmusportal-server
+rules:
+- apiGroups: [eventtracker.litmuschaos.io]
+  resources: [eventtrackerpolicies]
+  verbs: [create, delete, get, list, patch, update, watch]
+- apiGroups: [eventtracker.litmuschaos.io]
+  resources: [eventtrackerpolicies/status]
+  verbs: [get, patch, update]
+- apiGroups: ["", extensions, apps]
+  resources: [deployments, daemonsets, statefulsets, pods, configmaps]
+  verbs: [get, list, watch]
```

Comment:
this too can be at the agent level

Reply:
same-https://github.com/litmuschaos/litmus/pull/3017#discussion_r669711685